mod/state-transition/pkg/core/state_processor_withdrawals.go

```
@@ -79,6 +80,13 @@ func (sp *StateProcessor[
 
 	// Common validations
 	if len(expectedWithdrawals) != len(payloadWithdrawals) {
+		for i, wd := range expectedWithdrawals {
+			fmt.Printf("expectedWithdrawal %d: %s\n", i, spew.Sdump(wd))
```

golangci-lint (1.62.2)

use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)